cmd/status.go

```diff
@@ -176,16 +240,29 @@ func formatManifests(environment string, manifests []packages.PackageManifest) [
 			extraVersions = append(extraVersions, m.Version)
 		}
 	}
-	return formatManifest(environment, manifests[len(manifests)-1], extraVersions)
+	return formatManifest(environment, manifests[len(manifests)-1], extraVersions, extraParameters)
 }
 
 // formatManifest returns a row of data for a given package manifest
-func formatManifest(environment string, manifest packages.PackageManifest, extraVersions []string) []string {
+func formatManifest(environment string, manifest packages.PackageManifest, extraVersions []string, extraParameters []string) []string {
 	version := manifest.Version
 	if len(extraVersions) > 0 {
 		version = fmt.Sprintf("%s (%s)", version, strings.Join(extraVersions, ", "))
 	}
-	return []string{environment, version, releaseFromVersion(manifest.Version), manifest.Title, manifest.Description}
+
+	data := []string{environment, version, releaseFromVersion(manifest.Version), manifest.Title, manifest.Description}
+
+	for _, param := range extraParameters {
+		switch param {
+		case kibanaVersionParameter:
+			data = append(data, manifest.Conditions.Kibana.Version)
+		case categoriesParamter:
+			data = append(data, strings.Join(manifest.Categories, ","))
+		case elasticsearchSubscriptionParameter:
+			data = append(data, manifest.Conditions.Elastic.Subscription)
```

Comment:
Could we make this more generic? Maybe if we unpack the manifest in a `MapStr` we can `GetValue` anything.

Reply:
I've tried to use `MapStr` and it could be achieved something similar.
But I was not able to show a list of available parameters to use, so the usage for the users (with my tests) would be kind of difficult.
I would prefer to keep it as it is here for now.